controller/main_controller.php

```diff
@@ -93,7 +93,7 @@ public function display($route)
 	* @param string $route The route name for a page
 	* @return object $entity The entity object
 	* @throws http_exception
-	* @access public
+	* @access protected
```

Comment:
@VSEphpbb Honestly you shouldn't have to specify this annotation. It's coming from php4 ;)